tembo-cli/src/cmd/apply.rs

```diff
@@ -971,7 +971,7 @@ fn get_extensions_controller(
                 database: String::new(),
                 schema: None,
                 version: None,
-                enabled: extension.enabled,
+                enabled: extension.enabled.is_some(),
```

Comment:
I'm not quite following 🤔 

So if `extension.enabled` is `Some(false)`, it still gets set to `true`?

Reply:
I agree with vini, this should be unwrap or false

Reply:
Yes you're completely right. Changing it now